.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Pythonlint
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+
+jobs:
+  python-isort:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: ricardochaves/python-lint@v1.4.0
+        with:
+          python-root-list: "DentOS_Framework"
+          use-pylint: false
+          use-pycodestyle: false
+          use-flake8: false
+          use-black: false
+          use-mypy: false
+          use-isort: true
+          extra-isort-options: "--line-length 160 --wrap-length 80 --multi-line 3 --project dent_os_testbed --project gen --trailing-comma --force-grid-wrap 2"
+  
```

Comment:
The file ending is still not UNIX style.

Reply:
I checked on my side and it was reported as unix line endings, maybe git client on this machine does something funny.
![image](https://user-images.githubusercontent.com/32941360/211855607-4471ec97-cd78-4910-803b-1a366673d475.png)

maybe a github action to do dos2unix in all incoming files will be a great idea.